Classes/Other/Secrets.swift

```diff
@@ -14,3 +14,9 @@ public enum GithubAPI {
     static let clientSecret = "YOUR_CLIENT_SECRET"
 
 }
+
+public enum ImgurAPI {
+    
+    static let clientID = "<# Imgur API client ID #>"
```

Comment:
Why the `<# #>`? Would just keep this in line with the others (so `IMGUR_API_CLIENT_ID`)

Reply:
More as a forceful reminder for Ryan to populate the value when testing/releasing

Reply:
Hm, but the build doesn't complain even with the placeholder there. Which made me think - maybe we can print some warnings in debug mode when these are set to default?

Reply:
Huh!? Xcode 9 has always given me a compile error if there's a placeholder in code... I can give it a default value I'm not fussed - was intended to just be a reminder!

Reply:
Thought so as well, but seems to work fine. Maybe because it's in a `String`?

Reply:
Yep, `"<# test #>"` is just a string containing `"<# test #>"` at compile time it seems.

Reply:
Maybe 🤔 Today I learn